### PR TITLE
Databricks dolly v2 12b hftransformersv2

### PR DESCRIPTION
**Pull request by KonakanchiSwathi**
Databricks dolly v2 12b hftransformersv2

### CODE REVIEW

**Comment by KonakanchiSwathi** [comment]
Job link: -https://ml.azure.com/experiments/id/f868cd36-9ea7-400f-8830-5067656cb448/runs/dc71b237-96ed-4308-98e0-dc62dfdb37c6?wsid=/subscriptions/72c03bf3-4e69-41af-9532-dfcdc3eefef4/resourceGroups/shared-finetuning-rg/providers/Microsoft.MachineLearningServices/workspaces/v-swathikon&tid=72f988bf-86f1-41af-91ab-2d7cd011db47#
Deploy: - Ran it separately due to quota issue

File: assets/models/system/databricks-dolly-v2-12b/model.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 path:
   container_name: models
-  container_path: huggingface/databricks-dolly-v2-12b/7dde7ac/mlflow_model_folder
+  container_path: huggingface/databricks-dolly-v2-12b/1693844710/mlflow_model_folder
```
Comment (skasturi) [inline assets/models/system/databricks-dolly-v2-12b/model.yaml:3]: Space in the path?

File: assets/models/system/databricks-dolly-v2-3b/spec.yaml [hunk 2]
```diff
@@ -29,4 +29,4 @@ tags:
       
 
   task: text-generation
-version: 1
+version: 2
```
Comment (skasturi) [inline assets/models/system/databricks-dolly-v2-3b/spec.yaml:32]: Where can I find other files viz.; model.yaml, description etc?

**Review by skasturi — CHANGES_REQUESTED** [review]
Please address review comments.

**Comment by skasturi** [comment]
Could you please list all the changes being done in the PR, in the description? 

Title seems to talk about Dolly only & actual changes seem to be quite a few.

File: assets/models/system/databricks-dolly-v2-3b/spec.yaml (hunk 2 above)
Comment (KonakanchiSwathi) [reply assets/models/system/databricks-dolly-v2-3b/spec.yaml:32]: As we have added this model for first time, I have run 2 times this model one without Fine tune of Token classification and one with Finetune of Token classification.  

First PR of it is in  https://github.com/Azure/azureml-assets/pull/1161/commits/8f27344de0948d108b37f96c291255e6ba9bfbd6


Second PR is this that is why I changed the Version to 2

File: assets/models/system/databricks-dolly-v2-12b/model.yaml (hunk 1 above)
Comment (KonakanchiSwathi) [reply assets/models/system/databricks-dolly-v2-12b/model.yaml:3]: As I don't see the Space but Raised a PR for all in the new request.
https://github.com/Azure/azureml-assets/pull/1181

**Comment by KonakanchiSwathi** [comment]
Raised new PR for all Models at one place please find link below:-

https://github.com/Azure/azureml-assets/pull/1181/commits/e5799f19639d0e379a5ebcf8fa106e5dd641efa2

**Pull request closed without merging**